CONTRIBUTING.adoc

```diff
@@ -5,9 +5,9 @@ Here are few guidelines that should help you get started.
 
 == Using GitHub issues
 
-We use GitHub issues to track bugs and enhancements. Found a bug in the source code? You want to propose new features or enhancements  You can help us by submitting an issue in our https://github.com/gravitee-io/json-schema-generator-maven-plugin[repository]. Before you submit your issue, search the https://github.com/gravitee-io/json-schema-generator-maven-plugin/issues[issues archive] or the https://waffle.io/gravitee-io/release[backlog]; maybe your question was already answered.
```

Comment:
The issues are not centralized anymore ? https://github.com/gravitee-io/json-schema-generator-maven-plugin/issues

Reply:
yes it is, not sure to understand your comment

Reply:
Issues link should redirect to https://github.com/gravitee-io/issues/issues no ?

Reply:
oh ok I did not change this, I can do it

Reply:
fixed